generate_sbom

```diff
@@ -935,6 +942,18 @@ sub spdx_encode_header {
   return $spdx;
 }
 
+sub spdx_encode_dist {
+  my ($dist) = @_;
+
+  return spdx_encode_pkg({
+    NAME => $dist->{id},
+    VERSION => $dist->{version_id},
+    spdx_id => sprintf('SPDXRef-OperatingSystem-%s', gen_pkg_id($dist)),
+    primaryPackagePurpose => 'OPERATING-SYSTEM',
+    skip_external_refs => 1
+  }, undef, undef, {});
+
```

Comment:
I think it's more clean to not call spdx_encode_pkg but just do the stuff we need for the operating system entry here.

Reply:
Even if the filesystem entry is reflected _exactly_ as a package inside the OS?